### ISSUE TITLE
Fix wrong can message length

### ISSUE THREAD

**Issue by joaoantoniocardoso** [ZeniteSolar/MAB20#6]
All defined topics has its own length defined on [can_ids.h](https://github.com/ZeniteSolar/CAN_IDS/blob/master/can_ids.h).

See lines below:
https://github.com/ZeniteSolar/MAB20/blob/master/firmware/src/can_app.c#L57
https://github.com/ZeniteSolar/MAB20/blob/master/firmware/src/can_app.c#L77

Instead of `CAN_MSG_GENERIC_STATE_LENGTH`, it should be using `CAN_MSG_MAB19_STATE_LENGTH` and `CAN_MSG_MAB19_PUMPS_LENGTH`.

**Comment by s0sharma** [comment]
Hi, I would like to work on this.

**Comment by joaoantoniocardoso** [comment]
Hi @s0sharma, nice! We'll appreciate your contribution! To do so, you can follow the standard open source community workflow: 
- create a fork, 
- modify, 
- open a PR,

Please feel free to ask any question, thanks!

**Comment by joaoantoniocardoso** [comment]
Yeah, exactly, as simple as it sounds. Actually the same problem exists in
other modules too, but luckily it happens that the length for the generic
state message and the state message for all modules are the same, so,
nothing critical, it keeps working correctly even when the wrong macro was
used for state messages. No pressure here, but I would be very happy if you
fix all these:
https://github.com/search?q=org%3AZeniteSolar+CAN_MSG_GENERIC_STATE_LENGTH&type=code

Thanks again!
